src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java

```diff
@@ -51,19 +55,23 @@ public static void cleanupDotContent(String root, String lineEnding, String[] co
 						lines.add(line);
 					}
 				}
-				String line = null;
-				if (StringUtils.equals(lineEnding, UNIX)) {
-					line = UNIX_LINE_ENDING;
-				} else if (StringUtils.equals(lineEnding, DOS)) {
-					line = DOS_LINE_ENDING;
-				}
-				FileUtils.writeLines(file, CharEncoding.UTF_8, lines, line);
+				FileUtils.writeLines(file, CharEncoding.UTF_8, lines, getLineEnding(lineEnding));
```

Comment:
one thing is a bit confusing here

getLineEnding(lineEnding) - sounds funny if you read it outloud. 

maybe we should change the method name into something like getLineEndinchChar?